### PR TITLE
Show cards after moving into another list

### PR DESCRIPTION
**Pull request by luka-nextcloud**
Signed-off-by: Luka Trovic <luka@nextcloud.com>

* Resolves: #3676
* Target version: master 

### Summary
Cards disappear after moving from one list into another.

### TODO

- [ ] ...

### Checklist

- [x] Code is properly formatted
- [x] Sign-off message is added to all commits
- [ ] Tests (unit, integration, api and/or acceptance) are included
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: src/components/cards/CardMenu.vue [hunk 1]
```diff
@@ -167,10 +167,14 @@ export default {
 				},
 			})
 		},
-		moveCard() {
+		async moveCard() {
 			this.copiedCard = Object.assign({}, this.card)
+			const boardId = this.card?.boardId ? this.card.boardId : this.$route.params.id
```
Comment (vinicius73) [inline src/components/cards/CardMenu.vue:172]: it can be moved to a computed.

File: src/components/cards/CardMenu.vue [hunk 2]
```diff
@@ -167,10 +167,14 @@ export default {
...
 			this.copiedCard.stackId = this.selectedStack.id
 			this.$store.dispatch('moveCard', this.copiedCard)
```
Comment (vinicius73) [inline src/components/cards/CardMenu.vue:174]: Actions are async.

**Review by vinicius73 — APPROVED** [review]

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**